packages/fx-core/src/component/driver/arm/error/invalidParameterUserError.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { UserError } from "@microsoft/teamsfx-api";
+import { getDefaultString, getLocalizedString } from "../../../../common/localizeUtils";
+
+const errorCode = "InvalidParameter";
+const messageKey = "driver.arm.error.invalidParameter";
+
+export class InvalidParameterUserError extends UserError {
```

Comment:
You may reuse similar error in AAD driver (and move it out of AAD folder). The wording already considers supporting different drivers.

Reply:
It is fixed. Thanks